src/test/java/org/opensearch/ad/feature/NoPowermockSearchFeatureDaoTests.java

```diff
@@ -254,15 +255,14 @@ public void testGetHighestCountEntitiesUsingTermsAgg() {
             SearchFeatureDao.AGG_NAME_TOP,
             InternalOrder.key(false),
             BucketOrder.count(false),
-            1,
-            0,
             Collections.emptyMap(),
             DocValueFormat.RAW,
             1,
             false,
             0,
             stringBuckets,
-            0
+            0,
+            new TermsAggregator.BucketCountThresholds(1, 0, 1, 0)
```

Comment:
An extra field for extra tests? I see (1,0,0) being deleted -- and the addition is (1,0,1,0).

Reply:
This is due to changes in the core. Related wording in the PR description:

Updated StringTerms constructor. Removed requiredSize and minDocCount, added TermsAggregator.BucketCountThresholds. See details [here](https://github.com/opensearch-project/OpenSearch/commit/15b7de00f2ff75587560ac6e38ffb20da030178e)